deploy/internal/statefulset-postgres-db.yaml

```diff
@@ -19,92 +19,47 @@ spec:
         noobaa-db: postgres
     spec:
       serviceAccountName: noobaa-db
-      initContainers:
-      #-----------------#
-      # INIT CONTAINERS #
-      #-----------------#
-      - name: initialize-database
-        image: NOOBAA_DB_IMAGE
-        env:
-          - name: POSTGRESQL_DATABASE
-            value: nbcore
-          - name: LC_COLLATE
-            value: C
-          - name: POSTGRESQL_USER
-            valueFrom:
-              secretKeyRef:
-                key: user
-                name: noobaa-db
-          - name: POSTGRESQL_PASSWORD
-            valueFrom:
-              secretKeyRef:
-                key: password
-                name: noobaa-db
-        command:
-        - sh
-        - -x
-        - /init/initdb.sh
-        securityContext:
-          runAsUser: 0
-          runAsGroup: 0
-        resources:
-          requests:
-            cpu: "500m"
-            memory: "500Mi"
-          limits:
-            cpu: "500m"
-            memory: "500Mi"
-        volumeMounts:
-        - name: db
-          mountPath: /var/lib/pgsql
-        - name: noobaa-postgres-initdb-sh-volume
-          mountPath: /init
       containers:
```

Comment:
I think removing all the init containers will result in this code failing as it expects init_containers[0] to exist:
https://github.com/noobaa/noobaa-operator/blob/6ddc25e52832366289685c4d813c35642b920c09/pkg/system/phase2_creating.go#L1537